### ISSUE TITLE
Fix lint errors

### ISSUE THREAD

**Issue by krt16s** [xpheres/AndroidAnalyticalTranslator#8]
Compiling the app throws lint errors:

````
:app:lintVitalRelease/home/krt/src/fdroiddata/build/com.example.root.analyticaltranslator/app/src/main/res/layout/activity_main.xml:75: Error: Duplicate id @+id/textView4, already defined earlier in this layout [DuplicateIds]
            android:id="@+id/textView4" />
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /home/krt/src/fdroiddata/build/com.example.root.analyticaltranslator/app/src/main/res/layout/activity_main.xml:55: @+id/textView4 originally defined here

   Explanation for issues of type "DuplicateIds":
   Within a layout, id's should be unique since otherwise findViewById() can
   return an unexpected view.

1 errors, 0 warnings
 FAILED
````

I can ignore this on my side by adding `android { lintOptions { abortOnError false } }` to `app/build.gradle` but actually that should behandled by upstream.

**Comment by xpheres** [comment]
Solved: https://github.com/xpheres/AndroidAnalyticalTranslator/blob/master/app/src/main/res/layout/activity_main.xml

Now it should be fine

**Comment by krt16s** [comment]
Closed by b471dd24eea309b51ed675e2712d63bef16cf5de.
